tests/Control/UserDefinedFormControllerTest.yml

```diff
@@ -0,0 +1,340 @@
+      - =>SilverStripe\UserForms\Model\Recipient\EmailRecipient.no-data
+
+  page-with-group:
+    Content: 'Page with group'
+    Title: 'page with group'
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form7step1
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFieldGroup.group1start
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.some-field
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFieldGroupEnd.group1end
+
+  form-with-reset-and-custom-action:
+    Title: Form with Reset Action
+    SubmitButtonText: Custom Button
+    ShowClearButton: true
+
+  validation-form:
+    Title: Validation Form
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form3step1
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.required-text
+
+  custom-rules-form:
+    Title: Custom Rules Form
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableCheckbox.checkbox-2
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.basic-text-2
+
+  summary-rules-form:
+    Title: Summary Fields Form
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.summary-show
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.summary-hide
+
+  empty-form:
+    Title: Empty Form
+
+  filtered-form-page:
+    Title: 'Page with filtered recipients'
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form5step1
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableCheckboxGroupField.checkbox-group
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.your-name-field
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.street-field
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.city-field
+    EmailRecipients:
+      - =>SilverStripe\UserForms\Model\Recipient\EmailRecipient.unfiltered-recipient-1
+      - =>SilverStripe\UserForms\Model\Recipient\EmailRecipient.filtered-recipient-1
+      - =>SilverStripe\UserForms\Model\Recipient\EmailRecipient.filtered-recipient-2
+
+  empty-page:
+    Title: 'Page with empty step'
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form6step1
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.field-1
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form6step2
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.field-2
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableFormStep.form6step3
+
+  email-form:
+    Title: 'Page with email field'
+    Fields:
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableEmailField.another-email-field
+      - =>SilverStripe\UserForms\Model\EditableFormField\EditableTextField.another-required
```

Comment:
Is this a duplicate now? I'm expecting a file rename, or the original to be deleted